ui/components/component-library/avatar-badge/avatar-badge.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box/box';
+import { BaseAvatar } from '../base-avatar';
+
+export const badgePosition = ['top', 'bottom'];
+
+export const AvatarBadge = ({
```

Comment:
Is there any reason we changed the name from `AvatarWithBadge` to `AvatarBadge`?

Reply:
Ah Sorry! My bad. Changed it now